editor/editor_settings.cpp

```diff
@@ -703,6 +703,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("project_manager/sorting_order", 0);
 	hints["project_manager/sorting_order"] = PropertyInfo(Variant::INT, "project_manager/sorting_order", PROPERTY_HINT_ENUM, "Name,Path,Last Edited");
 
+	_initial_set("project_settings/confirm_setting_deletion", true);
```

Comment:
Do we even need the confirmation if deletion can be undone? (In `master` this does not work due to regression, see #45820.) Moreover, deletion confirmation is not used in other Project Settings tabs.

Reply:
If something can be undone, I don't think a confirmation is necessary. Also, this should not be a settings, either a confirmation is needed or not, but it should not be something configurable.